tests/detect-ldap-request-operation/test.yaml

```diff
@@ -0,0 +1,55 @@
+requires:
+  min-version: 8
+
+args:
+  - -k none
+
+pcap: input.pcap
+
+checks:
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        alert.signature_id: 1
```

Comment:
We want to test app-layer metadata logging also so that there is a `ldap.request.operation` or similar